Tests/DatadogTests/Datadog/RUM/WebView/WKUserContentController+DatadogTests.swift

```diff
@@ -50,12 +56,14 @@ class WKUserContentController_DatadogTests: XCTestCase {
 
         let initialUserScriptCount = controller.userScripts.count
 
-        controller.addDatadogMessageHandler(allowedWebViewHosts: ["datadoghq.com"], hostsSanitizer: mockSanitizer)
+        (0..<5).forEach { _ in
+            controller.addDatadogMessageHandler(allowedWebViewHosts: ["datadoghq.com"], hostsSanitizer: mockSanitizer)
+        }
```

Comment:
It's very unclear why we add this handler 5 times. If we want to test that _"given handler already registered, when registering new handler, it removes the previous one"_, this should be a separate test case.